schema/core/reusable/object_storage_container_data.json

```diff
@@ -0,0 +1,37 @@
+{
+    "schemaId": "object-storage-container-data",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Object Storage Container Data",
+    "properties": {
+        "CONTAINER": {
+            "description": "Container that ran the job resulting in the file",
+            "type": "string"
+        },
+        "NAME": {
+            "description": "Path to where the job put the file",
```

Comment:
Name of the file inside the object storage bucket

Reply:
Fixed in recent commit